package.json

```diff
@@ -39,36 +40,50 @@
     "@itwin/core-orbitgt": "4.10.3",
     "@itwin/core-quantity": "4.10.3",
     "@itwin/core-react": "4.17.6",
+    "@itwin/ecschema-metadata": "4.10.3",
+    "@itwin/eslint-plugin": "4.0.0-dev.48",
     "@itwin/imodel-components-react": "4.17.6",
     "@itwin/mobile-sdk-core": "0.22.19",
     "@itwin/presentation-common": "4.10.3",
     "@itwin/presentation-frontend": "4.10.3",
     "@itwin/webgl-compatibility": "4.10.3",
-    "classnames": "^2.2.6",
-    "react": "^17.0.0",
-    "react-dom": "^17.0.0",
-    "react-redux": "^7.2.4",
-    "redux": "^4.1.1"
-  },
-  "devDependencies": {
-    "@babel/core": "^7.15.8",
-    "@itwin/eslint-plugin": "4.0.0-dev.48",
     "@svgr/cli": "^6.3.1",
     "@types/node": "10.14.1",
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
     "@types/react-redux": "^7.1.19",
     "@typescript-eslint/eslint-plugin": "^5.55.0",
     "@typescript-eslint/parser": "^5.55.0",
+    "classnames": "^2.2.6",
     "cpx2": "^4.2.0",
     "eslint": "^8.36.0",
     "eslint-plugin-react-hooks": "^4.6.0",
+    "react": "^17.0.0",
+    "react-dom": "^17.0.0",
+    "react-redux": "^7.2.4",
+    "redux": "^4.1.1",
     "relative-deps": "^1.0.7",
     "sass": "^1.39.0",
     "ts-node": "^10.3.0",
     "typedoc": "~0.22.0",
     "typescript": "~5.0.2"
   },
+  "peerDependencies": {
+    "@bentley/icons-generic-webfont": "1.0.34",
+    "@itwin/appui-abstract": "4.10.3",
+    "@itwin/appui-react": "4.17.6",
+    "@itwin/core-bentley": "4.10.3",
+    "@itwin/core-common": "4.10.3",
+    "@itwin/core-frontend": "4.10.3",
+    "@itwin/core-geometry": "4.10.3",
+    "@itwin/core-react": "4.17.6",
+    "@itwin/imodel-components-react": "4.17.6",
+    "@itwin/mobile-sdk-core": "0.22.19",
+    "classnames": "^2.2.6",
```

Comment:
classnames should be pure and a direct dep

Reply:
Fixed